src/pcapng.h

```diff
@@ -27,43 +27,44 @@ Helpers for working with .pcapng files.
 
 #include <pshpack1.h>
 struct PCAPNG_BLOCK_HEAD {
-    long Type;
-    long Length;
+    DWORD Type;
```

Comment:
Please either undo these fixes and add the new structs with native C types, or fix the rest of the type names in the code to use Windows types (for instance, params and local variables are still being declared as ints and chars etc)